codespaces-start.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if ! spawnctl get data-images > /dev/null 2>&1 ; then
+  echo -e "$RED You must authenticate to the Spawn service before using this codespace environment $NC"
```

Comment:
I don't have `$RED` or `$NC` defined. `spawn.sh` defines these for functions that want to print in color, maybe that's what you need to do:

https://github.com/red-gate/spawn-demo/blob/main/spawn.sh#L53

Reply:
yep, I forgot that, nice catch